src/main/java/org/apache/datasketches/theta/UnionImpl.java

```diff
@@ -431,6 +426,10 @@ private void processVer3(final Memory skMem) {
   //has seedHash and p, could have 0 entries & theta,
   // can only be compact, ordered, size >= 8
   private void processVer2(final Memory skMem) {
+    final int famId = extractFamilyID(skMem);
+    if (famId != 3) {
```

Comment:
should this refer to a constant in Family?

Reply:
Fixed.